src/index.js

```diff
@@ -13,9 +13,9 @@ export default options => {
 
   /**
    * Get rooms socket is subscribed to.
-   *
-   * @param {String} id
-   * @param {Function} [fn]
+   * Used for get rooms
```

Comment:
It is sufficient, however following the [JSDoc](http://usejsdoc.org/tags-param.html) specs for description to ensure it is clearer/easier for people to jump in and understand what is going on